data_types.py

```diff
@@ -1,5 +1,8 @@
 import numpy as np
-from skimage import color
+from typing import List
+from skimage import img_as_ubyte, color
+
+UINT8_MAX_VALUE = 256
```

Comment:
isn't it 255?

Reply:
The name could have been a little confusing, changed it.

Reply:
The name could have been a little confusing, changed it.

Reply:
nice